### ISSUE TITLE
Dividend objects returned by list_dividends() don't have a "currency" attribute

### ISSUE THREAD

**Issue by bfoz** [polygon-io/client-python#494]
When fetching dividends using the python client (v1.12.2), the resulting objects are missing the "currency" field that the API documentation suggests should be there. 

Fetching from the Documentation page (or https://api.polygon.io/v3/reference/dividends?ticker=ENB&apiKey=) yields:
```
{
    "results": [
        {
            "cash_amount": 0.8875,
            "currency": "CAD",
            "declaration_date": "2023-08-01",
            "dividend_type": "CD",
            "ex_dividend_date": "2023-08-14",
            "frequency": 4,
            "pay_date": "2023-09-01",
            "record_date": "2023-08-15",
            "ticker": "ENB"
        },
       ...
```

However, using the python client yields:
```
from polygon import RESTClient
client = RESTClient(key)
dividends = list(client.list_dividends('ENB'))
print(dividends[0])
print(dividends[0].cash_amount)
print(dividends[0].currency)

Dividend(cash_amount=0.8875, declaration_date='2023-08-01', dividend_type='CD', ex_dividend_date='2023-08-14', frequency=4, pay_date='2023-09-01', record_date='2023-08-15', ticker='ENB')
0.8875
Traceback (most recent call last):
  File "...", line 12, in <module>
    print(dividends[0].currency)
          ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Dividend' object has no attribute 'currency'
```

**Comment by justinpolygon** [comment]
Hey @bfoz, we just fixed the bug and released v1.12.3. It might take a minute to roll out but you should see if this fixed if you update the client library via `pip install -U polygon-api-client`. Cheers and thanks for reporting this!

**Comment by bfoz** [comment]
That was fast! 

It looks good now:
```
Dividend(cash_amount=0.8875, currency='CAD', declaration_date='2023-08-01', dividend_type='CD', ex_dividend_date='2023-08-14', frequency=4, pay_date='2023-09-01', record_date='2023-08-15', ticker='ENB')
0.8875
CAD
```

Thanks!
